core/src/main/java/org/apache/iceberg/GenericManifestEntry.java

```diff
@@ -154,12 +140,6 @@ public void setSnapshotId(long newSnapshotId) {
   public void setDataSequenceNumber(long newDataSequenceNumber) {
     // always reset sequenceNumber whenever dataSequenceNumber is changed
```

Comment:
Shall we remove this comment as well? It no longer applies as we removed `sequenceNumber`.

Reply:
good call, removed it